contracts/staking/interfaces/IPoSValidatorManager.sol

```diff
@@ -39,6 +39,7 @@ struct Delegator {
 }
 
 struct PoSValidatorRequirements {
+    address owner;
```

Comment:
If we're adding `owner` here, we should probably name this something other than `requirements`